### PR TITLE
Issue #13345: enable examples check for OrderedPropertiesCheck 

### PR DESCRIPTION
**Pull request by lmgyuan**
Part of Issue https://github.com/checkstyle/checkstyle/issues/13345 

Hi, 
I am creating this PR so I can work on this and ask questions here, instead of spamming the public chat thread. May I ask whether I am doing it correctly by creating Example1.properties instead of Example1.java? My understanding is that this test is checking the order of properties in the properties file. Please comment if I did anything incorrectly. Any feedback is welcome!

### CODE REVIEW

**Comment by Lmh-java** [comment]
Please squash your commits to one single commit, otherwise CI will fail.

Also, please change your commit message. It must obey the following rules:
Proper commit message should adhere to the following rules:

    1) Must match one of the following patterns:
        ^Issue #\d+: .*$
        ^Pull #\d+: .*$
        ^(minor|config|infra|doc|spelling|dependency|supplemental): .*$
    2) It contains only one line of text
    3) Must not end with a period, space, or tab
    4) Commit message should be less than or equal to 200 characters

You can run `mvn clean verify` locally after your commit to your local repo to validate.

**Comment by Lmh-java** [comment]
> Part of Issue #13345
> 
> Hi, I am creating this PR so I can work on this and ask questions here, instead of spamming the public chat thread. May I ask whether I am doing it correctly by creating Example1.properties instead of Example1.java? My understanding is that this test is checking the order of properties in the properties file. Please comment if I did anything incorrectly. Any feedback is welcome!

Hi, you need to move the lines from `Example1.txt` to `Example1.java` (not the properties file). Since the new file is a Java file, you also need to make sure it can be complied with normal Java syntax. (Consider contain those lines in a method or something like that)

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -721,14 +721,6 @@
...
             files="checks[\\/]javadoc[\\/]javadocmissingleadingasterisk[\\/]InputJavadocMissingLeadingAsteriskCorrect.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValid.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValid.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]javadoc[\\/]javadocmissingwhitespaceafterasterisk[\\/]InputJavadocMissingWhitespaceAfterAsteriskValidWithTab.java"/>
   <suppress id="UnnecessaryOkComment"
```
Comment (MANISH-K-07) [inline config/checkstyle-input-suppressions.xml:724]: These changes belong to previously merged commits and are a result of improper rebase.

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/orderedproperties/Example1.properties [hunk 2]
```diff
@@ -0,0 +1,13 @@
+#xml
+#<module name="Checker">
+#<module name="OrderedProperties"/>
+#</module>
+
```
Comment (MANISH-K-07) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/orderedproperties/Example1.properties:5]: @lmgyuan , Please go through the issue description again. Refer successful PRs for better understanding.
You will have to make changes here.

**Review by MANISH-K-07 — COMMENTED** [review]
Items :

**Comment by romani** [comment]
we lost connection to contributor

**Pull request closed without merging**